### DISCUSSION TITLE
In FO listing pages, display the “Only X left in stock” label

### DISCUSSION THREAD

**Discussion by marionf** [PrestaShop/hummingbird#837] [Ideas]
#### Is your feature request related to a problem?

Currently, the remaining quantities are not displayed in FO product listings.

#### Describe the solution you'd like

As a visitor, I want to know easily when the last products are in stock when I am in a product listing.

The “only X left in stock” label is displayed in green #5A9166 in product listings if the default combination quantity is below the level defined.

Mockups: https://www.figma.com/file/aCBf2HlypjT0ZxP2XcsT68/Core-Github-issues?node-id=207%3A4875

<img width="812" alt="Capture d’écran 2021-09-24 à 17 45 57" src="https://user-images.githubusercontent.com/13449658/134704913-874bd0a7-3d60-4dbf-8af7-171a3b2fa8a1.png">

**Comment by Hlavtox** [comment]
This is something that is still missing from hummingbird guys. :-) @tblivet @ga-devfront

**Comment by ga-devfront** [comment]
@Hlavtox switched this issue inside https://github.com/PrestaShop/hummingbird and transform it into discussion. Thank you for your vigilance.
